### PR TITLE
Guide ie10 and ie11 fixes

### PR DESCRIPTION
**Pull request by darrenmason**
IE11 JS errors specific to guide are resolved.  

WIP: IE11 is not respecting width:100% and height:100% on container, still looking for an adequate workaround.

### CODE REVIEW

**Comment by philschatz** [comment]
Could you use something like http://getbootstrap.com/components/#responsive-embed on the `<svg>` instead of an `<iframe>`?

**Comment by darrenmason** [comment]
I used breakpoints that match the aspect ratio.  Resize now works in IE11.  This PR addresses IE10 as well.

File: resources/styles/components/learning-guide.less [hunk 1]
```diff
@@ -245,3 +249,151 @@
...
   }
 }
+
+
+
+// IE10/11 resize hack
+// these breakpoints correspond approx. with what height:auto calculates in other browsers.
+// only targets IE10 and above.
+
+@media screen and (max-width: 1079px) {
```
Comment (philschatz) [inline resources/styles/components/learning-guide.less]: Can you combine these into a single mixin? That way the breakpoints are each one line

**Comment by philschatz** [comment]
These changes were merged to master via #313

**Pull request closed without merging**